### DISCUSSION TITLE
玩家建议统计

### DISCUSSION THREAD

**Discussion by YuanYuanOwO** [YuanYuanOwO/Minecraft-Tianji-Server#23] [General]
- [ ] 服务器里做小游戏
- [ ] 更多附魔插件
- [ ] 新增空岛，小游戏 ，真实四季等玩法
- [ ] 更改传送等待CD、修改为传送冷却 
- [ ] 末影龙定期自动刷新调整
- [X] 随机传送范围调整
- [ ] 整合玩法插件及举办活动

**Comment by Qimukakash** [comment]
什么时候把789周目世界修一下呢，进不去了

**Comment by YuanYuanOwO** [comment]
> 什么时候把789周目世界修一下呢，进不去了

我会**试着**修一下的

**Comment by Qimukakash** [comment]
建议启用抽奖系统，抽取生存无法合成的道具（例如效率6，耐久4。。甚至是加移动速度之类的鞋子）用稀有矿物或者稀有道具，或者在原版中比较鸡肋的材料用作购买抽奖箱的货币，我觉得加入几种奖池之后，富有的玩家资金将大量被消耗，发展中玩家将更有动力去肝。

**Comment by YuanYuanOwO** [comment]
> 建议启用抽奖系统，抽取生存无法合成的道具（例如效率6，耐久4。。甚至是加移动速度之类的鞋子）用稀有矿物或者稀有道具，或者在原版中比较鸡肋的材料用作购买抽奖箱的货币，我觉得加入几种奖池之后，富有的玩家资金将大量被消耗，发展中玩家将更有动力去肝。

重复
#10

**Comment by Qimukakash** [comment]
> > 建议启用抽奖系统，抽取生存无法合成的道具（例如效率6，耐久4。。甚至是加移动速度之类的鞋子）用稀有矿物或者稀有道具，或者在原版中比较鸡肋的材料用作购买抽奖箱的货币，我觉得加入几种奖池之后，富有的玩家资金将大量被消耗，发展中玩家将更有动力去肝。
> 
> 重复 #10

奥重复了，没注意看，不好意思啊
